Azure/azure-cli
# [App Service] fix#-13035- Added validation for az webapp config access-restriction …

…to avoid adding duplicates ipaddress

**Description**  
#13035 


**Testing Guide**  


try to add the multiple rules with same Ip. Then it should not allow.

sample command: 

**az webapp config access-restriction add -g 13035RG -n 13035-testing --ip-address 208.131.0.0/16 -p 100** 

**History Notes**  


[AppService] - Added validation for az webapp config access-restriction to avoid adding duplicates IP rules 
---

This checklist is used to make sure that common guidelines for a pull request are followed.
